### PR TITLE
DIRECTOR: Detection improvements

### PR DESCRIPTION
**Pull request by BJNFNE**

### CODE REVIEW

File: engines/director/detection_tables.h [hunk 1]
```diff
@@ -9197,6 +9198,9 @@ static const DirectorGameDescription gameDescriptions[] = {
 	WINGAME2("babarcoins", "", "Start.exe",		"b198e9bf56c679747e4fd0972d5ba7b7", 2659300,
 							   "ba-01_A01.dxr", "cb38821091082aeb097e88551b791e05", 9946435, 1010),
 
+	// Actually Game use different Engine, Play.exe is something like an AutoRun.
+	WINGAME1_l("billibanniplay", "", "Play.exe", "t:6c95f5fa5fa9e107cc1e8a50491e262d", 3501155, Common::DE_DEU, 1010),
```
Comment (sev-) [inline engines/director/detection_tables.h:9202]: Then it is absolutely incorrect. The .exe file to provide must be the projector file.

File: engines/director/detection_tables.h [hunk 2]
```diff
@@ -2400,7 +2401,7 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACGAME1("cinevolt", "", "Cinema Volta", "2ce360c9ea2da80a2c6d1040b0ad92dd", 387911, 313),
 	WINGAME1("cinevolt", "", "CINEVOLT.EXE", "65d06b5fef155a2473434571aff5bc29", 370009, 313),
 
-	WINGAME1_l("cklasse", "", "c-klasse.exe", "39df1c7ff23ba627ddffaa5a77f83008", 376244, Common::DE_DEU, 313),
+	WINGAME1_l("cklasse", "v1.0", "c-klasse.exe", "39df1c7ff23ba627ddffaa5a77f83008", 376244, Common::DE_DEU, 313),
```
Comment (sev-) [inline engines/director/detection_tables.h:2404]: Were ther other versions released? If not, then it does not make sense to make this difference.

Comment (BJNFNE) [reply engines/director/detection_tables.h:2404]: This is the Demo from the ScummVM Director Demo site, when you exit the "Game" it says Version 1.0 from 1993, so i am not sure if its the only version that exists.

File: engines/director/detection_tables.h (hunk 1 above)
Comment (BJNFNE) [reply engines/director/detection_tables.h:9202]: Play.exe is the Autorun of the Game, The Autorun is using Director MX 2004, the Game itself uses an other Engine.

Comment (sev-) [reply engines/director/detection_tables.h:9202]: Is there any functionality in this autorun? Also, play.exe is way too generic, a second file needs to be added to the entry

File: engines/director/detection_tables.h (hunk 2 above)
Comment (sev-) [reply engines/director/detection_tables.h:2404]: Well, if it was the only one version release, there is no sense in adding this to the extra

File: engines/director/detection_tables.h (hunk 1 above)
Comment (BJNFNE) [reply engines/director/detection_tables.h:9202]: It starts the Game, it starts the installer. Like an normal autorun

Comment (bluegr) [reply engines/director/detection_tables.h:9202]: I agree with @sev-: I don't see the point of adding an installer to the detection entries

Comment (BJNFNE) [reply engines/director/detection_tables.h:9202]: Should i close then the PR? We have only one cklasse entry which have this version number as I am aware of it was version 1.0 Juni/July 1993, maybe i should move the information to the comments instead?

Comment (bluegr) [reply engines/director/detection_tables.h:9202]: If only one version of the game was released, then there's no point to specify it in the detection entry - both in the comments and the entry itself

**Comment by bluegr** [comment]
Thanks for your work @BJNFNE, however this pull request does not contain useful changes. Namely:

- The Reader Rabbit (Billi Bunny) series was developed by The Learning Company. I see no reference that they used Director in any of their games, so it's very strange that they chose to create an installer with Director. In any case, as mentioned, there's no point in adding entries for installers. For reference: https://www.mobygames.com/company/40/learning-company-the/games/
- Adding a version to a game that was only released with a single version is not providing any additional value

For the aforementioned reasons, I'm closing this pull request

**Pull request closed without merging**